src/app/fddev/tiles/fd_benchs.c

```diff
@@ -86,7 +86,7 @@ privileged_init( fd_topo_t *      topo,
 
   ushort port = 12000;
 
-  ctx->conn_cnt = fd_topo_tile_name_cnt( topo, "quic" );
```

Comment:
This change makes sense for the spammer, but not when running the existing bench command, since it should auto detect this from the topo still.

Reply:
Updated, plumbed it through using `layout.quic_tile_count`.